osc/osc.go

```diff
@@ -977,8 +977,14 @@ func writePaddedString(str string, buf *bytes.Buffer) (int, error) {
 		return 0, err
 	}
 
+	// Add a null terminator if not already present
+	if str[:len(str)-1] != 0 {
```

Comment:
That should be:

```go
if str[len(str)-1] != 0 {
```

Reply:
Which is exactly what I wrote in the playground, and then failed to write here :facepalm: